agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/localstorage/LocalFileSender.java

```diff
@@ -92,8 +94,10 @@ public void run() {
         resultCode.join(30, TimeUnit.SECONDS); // wait max 30 seconds for request to be completed.
       }
     } catch (RuntimeException ex) {
-      logger.error(
-          "Unexpected error occurred while sending telemetries from the local storage.", ex);
+      try (MDC.MDCCloseable ignored = AzureMonitorMsgId.DISK_PERSISTENCE_READ_ERROR.makeActive()) {
+        logger.error(
+            "Unexpected error occurred while sending telemetries from the local storage.", ex);
+      }
```

Comment:
i'm not sure anything is ever thrown above, have you seen this one in kusto?

Reply:
i found 2 occurrences from app service diagnostics kusto.